src/flow.ts

```diff
@@ -241,6 +242,14 @@ export class Flow {
     Tables.SOFTMAX_FACTOR = factor;
   }
 
+  static get UNISON(): boolean {
+    return Tables.UNISON;
+  }
+
+  static set UNISON(unison: boolean) {
+    Tables.UNISON = unison;
+  }
+
```

Comment:
Interesting. What does this flag do? It is only referenced on one line in the repo. Maybe in a different PR we can figure out which class should store this flag? Is it Metrics related, or is it StaveNote related?

Reply:
I guess Flow is fine for now....

Reply:
Left unmodified